flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/expressions/RexNodeConverter.java

```diff
@@ -865,6 +865,9 @@ private static Calendar valueAsCalendar(Object value) {
 
 	@Override
 	public RexNode visit(FieldReferenceExpression fieldReference) {
+		// We can not use inputCount+inputIndex+FieldIndex to construct field of calcite.
```

Comment:
@dawidwys @twalthr What do you think about this question? Should it be a bug of `QueryOperationConverter`? But I don't find any feasible in `RelBuilder`. Maybe we should keep same order in `AggregateQueryOperation` or adjust the order of existed `OperationTree`? Both ways look hard...